feat(service): treat html style attributes as CSS (#11080)
biomejs/biome: .changeset/feat-html-style-attribute-css.md

---
"@biomejs/biome": patch
---

> HTML `style` attribute values are now parsed as CSS. Biome reports CSS syntax errors in these attributes without changing their contents during formatting.

Reviewer: ```suggestion
HTML `style` attribute values are now parsed as CSS. All Biome CSS lint rules are applied to the `style` attributes.
```
